### DISCUSSION TITLE
Update failed

### DISCUSSION THREAD

**Discussion by themrwylde** [nextcloud/all-in-one#5101] [Questions] [answered]
Firstly, sorry. I didn't realise that we needed to update all of these containers (haven't done it in 10 months) and after upgrading to 9.3—the update failed so I rolled back to a backup and everything works until the NExtcloud container restarts. Then it breaks with the following:

Initializing nextcloud 29.0.4.1 ...
Initializing finished
Upgrading nextcloud from 28.0.0.11 to 29.0.4.1...

Then it fails...

I've tried:
Deleting the update.failed file and upgrading but it says it's already the latest
Deleting and reloading the Mastercontainer
Running Maintenance:repair etc etc

Any ideas? Should we just jump on the beta channel? Sorry if this is an obvious fix mate 😔

**Comment by szaimen** [comment, marked as answer]
Lets handle this in https://github.com/nextcloud/all-in-one/issues/5102
